core/app_modules/inventory.js

```diff
@@ -293,9 +292,9 @@ Object.defineProperty(inventory, 'checked_out', {
     var results = (yield client.query(query)).rows;
 
     var formatted = results.map(function(a) {
-      a.due = moment(a.due).toDate();
       a.call_number = a.call;
-      a.overdue = moment(a.due).isBefore(new Date());
+      a.due = new Date(a.due.setHours((new Date().getTimezoneOffset() > 0) ? 24 : 0)); //Compensates for bug created by being in different hemisphere
+      a.overdue = moment(a.due).isBefore(new Date().setHours(0, 0, 0, 0));
```

Comment:
Yeah, I guess it's not very intuitive, but isBefore() is false if they're the same, as they are in this case.